src/clear.js

```diff
@@ -12,4 +12,8 @@ const clearAll = () => {
       deleteTask(e);
     }
   });
+  let count = 0;
+  const data = Array.from(localData).filter((e) => e.completed === false);
+  data.map((e) => e.index = count += 1);
+  setStorage(data);
```

Comment:
- [ ] When I reload the page and try to change the checkbox of a task, the completed value of all the other tasks changes to false in the local storage. Please correct that.